Calculate.Tests/Calculate.Tests.csproj

```
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+		<IsPackable>false</IsPackable>
+		<IsTestProject>true</IsTestProject>
+	</PropertyGroup>
+
+	<ItemGroup>
```

If you use the IntelliTect tool test I mention in Program
```suggestion
	<ItemGroup>
	<PackageReference Include="IntelliTect.TestTools.Console" Version="1.0.0" />
```